.github/workflows/build-fluentbit-image.yaml

```diff
@@ -0,0 +1,261 @@
+        with:
+          platforms: linux/amd64,linux/arm64
+    
+      - name: Login to GHCR
+        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
+        with:
+          registry: docker.io
+          username: ${{ secrets.REGISTRY_USER }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+
+      - name: Build and push image
+        id: build
+        uses: docker/build-push-action@471d1dc4e07e5cdedd4c2171150001c434f0b7a4 # v6.15.0
+        with:
+          context: .
+          file: cmd/fluent-watcher/fluentbit/Dockerfile.debug
+          platforms: ${{ matrix.platform }}
+          labels: ${{ needs.build-tags.outputs.labels }}
+          provenance: false
+          build-args:
+            FLUENT_BIT_BASE_VERSION=${{ needs.build-info.outputs.version }}
+          outputs: type=image,"name=${{ env.GHCR_REPO }},${{ env.DOCKERHUB_REPO }}",push-by-digest=true,name-canonical=true,push=true
+
+      - name: Output image digests
+        id: output-digests
+        run: |
+          platform="${{ matrix.platform }}"
+          # Convert "linux/amd64" to just amd64 for the output variable name
+          arch=${platform#linux/}
+          echo "${arch}=${{ steps.build.outputs.digest }}" >> $GITHUB_OUTPUT
+
+  manifest:
+    name: Publish image manifests
+    runs-on: ubuntu-latest
+    needs: [build, build-debug,build-tags]
+    steps:
+      - name: Login to GHCR
+        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
+        with:
+          registry: docker.io
+          username: ${{ secrets.REGISTRY_USER }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+
+      - name: Create image manifest
+        uses: int128/docker-manifest-create-action@v2
+        with:
+          push: true
+          tags: ${{ needs.build-tags.outputs.tags }}
+          sources: | 
+            ${{ env.GHCR_REPO }}@${{ needs.build.outputs.digest_amd64 }}
+            ${{ env.GHCR_REPO }}@${{ needs.build.outputs.digest_arm64 }}
+
+      - name: Create debugimage manifest
+        uses: int128/docker-manifest-create-action@v2
+        with:
+          push: true
+          tags: ${{ needs.build-tags.outputs.debug-tags }}
+          sources: | 
+            ${{ env.GHCR_REPO }}@${{ needs.build-debug.outputs.digest_amd64 }}
+            ${{ env.GHCR_REPO }}@${{ needs.build-debug.outputs.digest_arm64 }}                        
+     
```

Comment:
This thing would change with `.editorconfig`. Maybe good idea to install the plugin for your IDEA.